README.md

```diff
@@ -150,6 +150,15 @@ http_request_duration_seconds_sum{handler="metrics",method="get"} 0.021113919999
 http_request_duration_seconds_count{handler="metrics",method="get"} 30
 ```
 
+kube-state-metrics exposes the build info metric:
+```
+kube_state_metrics_build_info{branch="master",goversion="go1.15.3",revision="6c9d775d",version="v2.0.0-beta"} 1
+```
+
+`kube_state_metrics_build_info` is used to expose version and other build information. For more usage about the info pattern,
+please check the blog post [here](https://www.robustperception.io/exposing-the-software-version-to-prometheus).
```

Comment:
I am not sure we are allowed to link to external companies blog posts? Otherwise, agreed this blog is very useful! @brancz wdyt?

Reply:
I don't think there is a general rule here, if this was self-promotion I would be more critical, but here it's really just a helpful resource.

Reply:
Yes sounds good, was on the edge. lgtm apart from the need to squash commits.